<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceListWidget</class>
 <widget class="QWidget" name="DeviceListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device list</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="connectDevice">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectDevice">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showLog">
       <property name="text">
        <string>Show log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewDump">
       <property name="text">
        <string>View dump</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="ipaddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="maxLength">
        <number>21</number>
       </property>
       <property name="placeholderText">
        <string>ipaddress[:port]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discoverDevice">
       <property name="text">
        <string>Discover device</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtIcons.qrc"/>
 </resources>
 <connections/>
</ui>
